app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterFragmentView.kt

```diff
@@ -0,0 +1,54 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.mvp
+
+import android.view.View
+import com.puzzlebench.clean_marvel_kotlin.R
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+import com.puzzlebench.clean_marvel_kotlin.presentation.CharacterFragment
+import com.puzzlebench.clean_marvel_kotlin.presentation.extension.getImageByUrl
+import com.puzzlebench.clean_marvel_kotlin.presentation.extension.showToast
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.dialog_response.*
+import java.lang.ref.WeakReference
+
+const val EMPTY_FIELD = ""
```

Comment:
private const should be in companion object

Reply:
Done!